### PR TITLE
docs: Update suites-I to parameterization.

### PR DESCRIPTION
**Pull request by arjclark**
This replaces the Jinja2 looping part of suites-I with parametrization - the now preferred way of doing that.

Jinja2 is kept in but with a more relevant `if` block example.

A more more in-depth parameterization tutorial can be done in a follow up pull request if we feel this example and those given in the cylc user guide are insufficient (the cylc ones are pretty good tbh for any advanced usage of pure parameterization - I'm tempted to do a blended Jinja2 and parameters one though).

@matthewrmshin - please review 1 and make sure you're happy with the new baseline advice
@oliver-sanders - please review 2 (but wait till Matt's happy with the general gist first)

### CODE REVIEW

**Comment by matthewrmshin** [comment]
(The test failure has nothing to do with this change.)

File: doc/rose-rug-suites-I.html [hunk 1]
```diff
@@ -586,40 +586,64 @@ <h3 id="dependencies-cycling">Cycling
...
-                to generate or insert text that will be expanded at
-                runtime.</p>
+                <p>This allows you to reduce down sections of the
+                <samp>suite.rc</samp> file into special instruction code that
+                will be expanded by cylc at runtime.</p>
               </div>
 
               <div class="slide">
-                <h3 class="alwayshidden">Jinja2 Loop</h3>
+                <h2 id="parameterized-graphing">Parameterized graphing</h2>
```
Comment (oliver-sanders) [inline doc/rose-rug-suites-I.html]: `<h3>` ?

File: doc/rose-rug-suites-I.html [hunk 2]
```diff
@@ -586,40 +586,64 @@ <h3 id="dependencies-cycling">Cycling
...
-{%- for WORLD in ["eris", "pluto", "makemake", "haumea"] %}
-  [[hello_{{ WORLD }}]]
+[scheduling]
+   [[dependencies]]
+         graph = hello_&lt;world&gt;
+</pre>
+              </div>
+
+              <div class="slide">
+                <h2 id="parameterized-runtime">Parameterized runtime</h2>
```
Comment (oliver-sanders) [inline doc/rose-rug-suites-I.html]: `<h3>` ?

File: doc/rose-rug-suites-I.html [hunk 3]
```diff
@@ -586,40 +586,64 @@ <h3 id="dependencies-cycling">Cycling
...
+                <p>Similarly, you can work with parameterized items in the
+                runtime section too, further helping you reduce down your
+                <samp>suite.rc</samp>:</p>
+                <pre class="prettyprint lang-cylc">
+
```
Comment (oliver-sanders) [inline doc/rose-rug-suites-I.html]: Blank line.

**Pull request merged**